File: Pertemuan 7/2112500851_AplikasiGabungan_AF/app/src/main/java/com/akbarrahmatm/a2112500851_aplikasigabungan_af/MainActivity.java
package com.akbarrahmatm.a2112500851_aplikasigabungan_af;

import androidx.appcompat.app.AppCompatActivity;

import android.content.Intent;
import android.os.Bundle;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btnmap = (Button) findViewById(R.id.btnmap);
        Button btnscroll = (Button) findViewById(R.id.btnscroll);

        btnmap.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, MapsActivity.class));
            }
        });

        btnscroll.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                startActivity(new Intent(MainActivity.this, ScrollViewActivity.class));
            }
        });
    }
}

File: Pertemuan 3/2112500851_Intent/app/src/main/java/com/akbarrahmatm/a2112500851_intent/MainActivity.kt
package com.akbarrahmatm.a2112500851_intent

import android.content.Intent
import android.net.Uri
import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import android.widget.Button

class MainActivity : AppCompatActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btn_implicit : Button = findViewById(R.id.implicit_intent)
        val btn_explicit : Button = findViewById(R.id.explicit_intent)

        btn_explicit.setOnClickListener{
            intent = Intent(this, SecondActivity::class.java)
            startActivity(intent)
        }
        btn_implicit.setOnClickListener {
            intent = Intent(Intent.ACTION_VIEW, Uri.parse("https://student.budiluhur.ac.id"))
            startActivity(intent)
        }

    }
}